docs/stubs/parsl.providers.error.OptionalModuleMissing.rst

```diff
@@ -1,6 +0,0 @@
-parsl.providers.error.OptionalModuleMissing
```

Comment:
Looks like `OptionalModuleMissing` in provider is removed. So this file should be removed as well?

Reply:
which file? `parsl.providers.error.OptionalModuleMissing.rst` *is* removed.

Reply:
Okay. I missed that.